src/database/schema/ChannelListSchema.ts

```diff
@@ -25,7 +26,7 @@ class ChannelList implements BaseDbSchema {
 
   unreadCount: number;
 
-  channelType: string;
+  channelType: BetterSocialChannelType;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Replacing `channelType: string;` with `channelType: BetterSocialChannelType;` is a good move towards type safety. It ensures that only valid channel types can be assigned to `channelType`.
```diff
-  channelType: string;
+  channelType: BetterSocialChannelType;
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->